coco/endpoint.py

```diff
@@ -236,6 +244,95 @@ def __init__(self, name, conf, slacker, forwarder, state):
 
         return result
 
+    def _save_reply(self, reply, path):
+        """
+        Save a forward call reply to state.
+
+        The replies of different hosts get merged.
+
+        Parameters
+        ----------
+        reply : dict
+            Keys are hosts and values are tuples of replies (dict) and HTTP status codes.
+        """
+        merged = dict()
+        for r in reply.values():
+            merged.update(r[0])
```

Comment:
won't you effectively be getting just the reply from the last host returned by `reply.values()`? What if a non-zero number of hosts return a failure, would you want to save the successful or failed replies? For the GPS endpoint I guess this is irrelevant since there is only one host involved, and I don't know of any other specific cases when this would be used, so maybe we can figure it out whenever the need comes up.

Reply:
Yes, in the case of thew GPS server I wouldn't want a structure like
```
gps_time/<host_name>/<result>
```
so I made it merge the replies from hosts for now (using `update()`). When we want to have a different behavior additionally in the future, we will have to add another config option.